docker/jenkins/build.sh

```diff
@@ -39,7 +39,7 @@ fi
 # build
 make -j${NPROC}
 # run the unit tests
-ctest -j${NPROC} --no-compress-output -T Test
+ctest -j1 --no-compress-output -T Test
```

Comment:
@Rombur This is all I could come up with for now

I think the timeout are related to #375

Reply:
I don't see how that makes sense that #375 is the problem. If you need to do that it means we cannot run two instances of DTK at the same time. So it could be that there is something that's acting like global variable in the code.

Reply:
Trying to set the `OMP_NUM_THREADS` env variable instead...